diff --git a/app/Console/Commands/AssignOrderToRiderAndMerchant.php b/app/Console/Commands/AssignOrderToRiderAndMerchant.php
index d4d5ee4..6503ee7 100644
--- a/app/Console/Commands/AssignOrderToRiderAndMerchant.php
+++ b/app/Console/Commands/AssignOrderToRiderAndMerchant.php
@@ -738,7 +738,7 @@ public function handle()
                 $merchantJobExists = AssignJob::where('order_id', $order->id)
                     ->where('code', OrderStatus::MERCHANT_PENDING_FOR_ACCEPTANCE)
                     ->exists();
-                if ($first_check && $total_riders == 0 && $total_merchants == 0 && !$riderJobExists && !$merchantJobExists) {
+                if ($first_check && $total_riders == 0 && $total_merchants == 0 && !$riderJobExists && !$merchantJobExists && !$order->is_pending_assign) {
 
                     // set to admin pending assign
                     $order->is_pending_assign = true;
